src/main/java/edu/ie3/datamodel/io/source/influxdb/InfluxDbWeatherSource.java

```diff
@@ -50,7 +54,7 @@ public InfluxDbWeatherSource(
     this.connector = connector;
     this.coordinateSource = coordinateSource;
     this.weatherValueFactory = weatherValueFactory;
-    this.coordinateIdColumnName = weatherValueFactory.getCoordinateIdFieldString();
+    this.coordinateIdFieldName = weatherValueFactory.getCoordinateIdFieldString();
```

Comment:
why do we track that here as an attribute explicitly. If it's just the value of the WeatherFactory we can just access it via the factory